src/middlewared/middlewared/plugins/kerberos.py

```diff
@@ -663,6 +665,31 @@ class Config:
                 'Failed to start kerberos service after deleting keytab entry: %s' % e
             )
 
+    @accepts(Dict(
+        'keytab_data',
+        Str('name', required=True),
+    ))
+    @job(lock='upload_keytab', pipes=['input'], check_pipes=False)
+    async def upload_keytab(self, job, data):
+        """
+        Upload a keytab file. This method expects the keytab file to be uploaded using
+        the /_upload/ endpoint.
+        """
+        job.check_pipe("input")
```

Comment:
Sorry, just noticed this, I think you can use check_pipes=True in the job decorator and remove this. Right, @themylogin ?

Reply:
Yes